sdk/src/main/java/com/hedera/hashgraph/sdk/TokenUpdateTransaction.java

```diff
@@ -158,6 +159,16 @@ public TokenUpdateTransaction setSupplyKey(Key key) {
         return this;
     }
 
+    public Key getCustomFeeKey() {
```

Comment:
Can we use `feeScheduleKey` instead of  `customFeeKey`. I feel like the former reads just a bit better. Up for debate though as long as we're consistent with the other SDKs.